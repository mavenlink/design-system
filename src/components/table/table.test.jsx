import React from 'react';
import { render } from '@testing-library/react';
import {
  Table,
  TableHeader,
  TableBody,
  TableRow,
  TableHeaderCell,
  TableCell,
} from './index.js';

test('rendering', () => {
  render((
    <Table>
      <TableHeader>
        <TableHeaderCell>id</TableHeaderCell>
        <TableHeaderCell>name</TableHeaderCell>
      </TableHeader>
      <TableBody>
        <TableRow>
          <TableCell>1</TableCell>
          <TableCell>Bob Ross</TableCell>
        </TableRow>
        <TableRow>
          <TableCell>2</TableCell>
          <TableCell>Bob Marley</TableCell>
        </TableRow>
      </TableBody>
    </Table>
  ));
  expect(document.body).toMatchSnapshot();
});
