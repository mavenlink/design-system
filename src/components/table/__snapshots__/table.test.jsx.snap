// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<body>
  <div>
    <table
      class="table"
    >
      <thead
        class="head"
      >
        <tr>
          <th
            class="headerCell"
          >
            id
          </th>
          <th
            class="headerCell"
          >
            name
          </th>
        </tr>
      </thead>
      <tbody
        class="body"
      >
        <tr
          class="row"
        >
          <td
            class="cell"
          >
            1
          </td>
          <td
            class="cell"
          >
            Bob Ross
          </td>
        </tr>
        <tr
          class="row"
        >
          <td
            class="cell"
          >
            2
          </td>
          <td
            class="cell"
          >
            Bob Marley
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</body>
`;
